mlos_bench/mlos_bench/services/local/local_exec.py

```diff
@@ -195,10 +195,8 @@ def _local_exec_script(self, script_line: str,
         cmd = [token for subcmd in subcmds for token in subcmd]
 
         env: Dict[str, str] = {}
-        # Need to include at least some basic environment variables to run the script.
-        env["PATH"] = environ["PATH"]
         if env_params:
-            env.update({key: str(val) for (key, val) in env_params.items()})
+            env = {key: str(val) for (key, val) in env_params.items()}
```

Comment:
Can you give an example of where this breaks?

Without this, things like this can fail when it needs to reference a python environment (set via the `PATH`) with `pandas` installed, for instance.

```python
#!/usr/bin/env python
import pandas
print("Hello World!")
```

Reply:
currently it breaks in all my environments, i.e. Windows and WSL/DevContainer on both of my PCs. On Windows, `PATH` is not in `environ` at all, and all unit tests with local_exec crash with `KeyError`.

In my DevContainers, only `launcher_run_test.py` tests fail, and it looks like the local_exec gets incorrect path or other environment variables when we launch `run.py` from inside the unit tests.

Rolling back this change fixes both issues.